test/unit/parser/to-collector.spec.js

```diff
@@ -127,76 +132,106 @@ function getParticipants(code, withStarter) {
+    expect(participants.GetPositions("_STARTER_")).toBeUndefined();
   });
 
-  test('A B->A.m', () => {
-    let rootContext = RootContext('A B B->A.m');
+  test("A B->A.m", () => {
+    let rootContext = RootContext("A B B->A.m");
     const participants = ToCollector.getParticipants(rootContext, true);
     expect(participants.Size()).toBe(2);
-    expect(participants.Get('_STARTER_')).toBeUndefined();
-    expect(participants.Get('B').isStarter).toBeTruthy();
-    expect(participants.Names()).toStrictEqual(['B', 'A']);
+    expect(participants.Get("_STARTER_")).toBeUndefined();
+    expect(participants.Get("B").isStarter).toBeTruthy();
+    expect(participants.Names()).toStrictEqual(["B", "A"]);
   });
 });
 
-describe('implicit', () => {
-  describe('from new', () => {
-    test('from new', () => {
-      let participants = getParticipants('new A()', true);
-      expect(participants.Get('A')).toEqual({
+describe("implicit", () => {
+  describe("from new", () => {
+    test("from new", () => {
+      let participants = getParticipants("new A()", true);
+      expect(participants.Get("A")).toEqual({
+        color: undefined,
+        comment: undefined,
         explicit: undefined,
         groupId: undefined,
-        name: 'A',
+        isStarter: false,
+        label: undefined,
+        name: "A",
         stereotype: undefined,
+        type: undefined,
         width: undefined,
       });
     });
-    test('seqDsl should treat creation as a participant - assignment', () => {
-      let participants = getParticipants('a = new A()', true);
+    test("seqDsl should treat creation as a participant - assignment", () => {
+      let participants = getParticipants("a = new A()", true);
       expect(participants.Size()).toBe(2);
-      expect(participants.Get('a:A').width).toBeUndefined();
+      expect(participants.Get("a:A").width).toBeUndefined();
+      expect(participants.GetPositions("a:A")).toEqual(new Set(["[8,9]"]));
     });
-    test('seqDsl should treat creation as a participant - assignment with type', () => {
+    test("seqDsl should treat creation as a participant - assignment with type", () => {
       // We need @Starter, otherwise IA becomes a participant declaration
-      let participants = getParticipants('@Starter(X) IA a = new A()', true);
+      let participants = getParticipants("@Starter(X) IA a = new A()", true);
+      expect(participants.Size()).toBe(2);
+      expect(participants.Get("X").width).toBeUndefined();
+      expect(participants.Get("a:A").width).toBeUndefined();
+      expect(participants.GetPositions("X").size).toBe(1);
+      expect(participants.GetPositions("X")).toEqual(new Set(["[9,10]"]));
+      expect(participants.GetPositions("a:A")).toEqual(new Set(["[23,24]"]));
+    });
+    test("seqDsl should treat method call as a participant - creation & assignment", () => {
+      let participants = getParticipants("a = new A()", true);
```

Comment:
This test is a dup of "seqDsl should treat creation as a participant - assignment".

Reply:
Removed